webdriver/tests/support/fixtures.py

```diff
@@ -123,6 +123,21 @@ def configuration():
     }
 
 
+async def reset_current_session_if_necessary(caps, bidi):
```

Comment:
Maybe we could rename the argument to `request_bidi`?

Reply:
Done, would you be able to merge this? Thanks!